src/daemon/rpmostreed-os.c

```diff
@@ -971,6 +971,14 @@ os_handle_automatic_update_trigger (RPMOSTreeOS *interface,
       g_assert_not_reached ();
     }
 
+  /* if redirect-output is not explicitly set, default to TRUE */
+  g_autoptr(GVariant) new_dict = NULL;
+  if (!g_variant_dict_contains (&dict, "redirect-output"))
+    {
+      g_variant_dict_insert (&dict, "redirect-output", "b", TRUE);
+      arg_options = new_dict = g_variant_ref_sink (g_variant_dict_end (&dict));
```

Comment:
Not pretty, but I couldn't bring myself to add yet another parameter to `os_merge_or_start_deployment_txn` for this. This seems in line with efforts to pass the many options we have through a `GVariant`.